### ISSUE TITLE
Recomendation for train without mobilenet pretrained

### ISSUE THREAD

**Issue by thancaocuong** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#79]
Thank for your awesome work. I have one question about how can I train the model without mobilenet pretrained? I mean I need to modify some piece of codes (some layers). So how can I choose the learning rate and weight decay? thank you!

**Comment by Daniil-Osokin** [comment]
Hi! Recent papers on pose estimation uses 1e-3 learning rate as starting point. You can use it with [ReduceLROnPlateau](https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau) scheduler. The general rule is to start from the biggest learning rate while network converges, then drop it properly. Usually it is a matter of experiments, but you can use 4e-5 weights decay for all layers. However, for depthwise conv layers in MobileNet v1 weight decay was equal to 0, because they already enough regularized by their number (significantly less than in standard convolutional layer).

**Comment by thancaocuong** [comment]
thank you for your recomendation!
